schemamd/render.go

```diff
@@ -161,7 +158,13 @@ func writeBlockChildren(w io.Writer, parents []string, block *tfjson.SchemaBlock
 		childBlock := block.NestedBlocks[n]
 		childAtt := block.Attributes[n]
 		for i, gf := range groupFilters {
-			if gf.filter(childBlock, childAtt) {
+			if n == "id" && childAtt.Description == "" {
+				if gf.name == "Read Only" {
```

Comment:
Instead of creating a new field `name` for `group filter`, just to support this comparison, wouldn't it be easier to just do a string contains `Read-Only`?